src/pages/ProductDetails.vue

```diff
@@ -161,20 +161,75 @@
         <div v-if="userAuthenticated()">
           <q-separator />
 
-          <q-card-actions class="row justify-end q-gutter-sm q-pr-sm">
+          <q-card-actions class="row justify-between q-gutter-md q-pr-md">
+            <q-btn
+              @click="enablePlatforms()"
+              color="primary"
+              style="width: 30%"
+              :loading="loadingPlatform"
+              no-caps
+            >
+              Add platforms
+            </q-btn>
             <q-skeleton type="circle" v-if="loadingPage" />
             <q-btn
               v-else
               color="negative"
               icon="delete"
               round
               @click="confirm = true"
+              :disable="loadingPlatform"
               :loading="loading"
             >
               <q-tooltip> Delete product </q-tooltip>
             </q-btn>
           </q-card-actions>
```

Comment:
Please do something like this:
```suggestion
          <q-card-actions class="justify-end q-pr-sm">
            <div class="q-gutter-md">
              <q-skeleton type="circle" v-if="loadingPage" />
              <q-btn
                @click="enablePlatforms()"
                color="primary"
                :loading="loadingPlatform"
                round
                icon="format_list_bulleted_add"
              >
                <q-tooltip> Add platforms to product </q-tooltip>
              </q-btn>
              <q-skeleton type="circle" v-if="loadingPage" />
              <q-btn
                v-else
                color="negative"
                icon="delete"
                round
                @click="confirm = true"
                :disable="loadingPlatform"
                :loading="loading"
              >
                <q-tooltip> Delete product </q-tooltip>
              </q-btn>
            </div>
          </q-card-actions>
```